docs/rfc/active/tags/README.md

```diff
@@ -0,0 +1,153 @@
+- Start Date: 2021-02-17
+- RFC PR: (after opening the RFC PR, update this with a link to it and update the file name)
+- Discussion Issue: (GitHub issue this was discussed in before the RFC, if any)
+- Implementation PR(s): (leave this empty)
+
+# Tags
+
+## Summary
+
+Adding general support for tag lowers the bar to add simple metadata to entities. Tags can be used as part of the
+business logic (e.g. sensitivity tag on a data set) or more ad hoc use cases, such as a data analyst's favorite
+dashboard. A general tag implementation will also allow us to define and search based on a new kind relationship, like
+which datasets and ML models are marked to contain PII data.
+
+> needs work
+
+## Basic example
+
+> If the proposal involves a new or changed API, include a basic code example. Omit this section if it's not applicable.
+
+## Motivation
+
+> Why are we doing this? What use cases does it support? What is the expected outcome?
+>
+> Please focus on explaining the motivation so that if this RFC is not accepted, the motivation could be used to develop
+> alternative solutions. In other words, enumerate the constraints you are trying to solve without coupling them too
+> closely to the solution you have in mind.
+
+## Requirements
+
+> What specific requirements does your design need to meet? This should ideally be a bulleted list of items you wish to
+> achieve with your design. This can help everyone involved (including yourself!) make sure your design is robust enough
+> to meet these requirements.
+>
+> Once everyone has agreed upon the set of requirements for your design, we can use this list to review the detailed
+> design.
+
+### Extensibility
+
+> Please also call out extensibility requirements. Is this proposal meant to be extended in the future? Are you adding a
+> new API or set of models that others can build on in later? Please list these concerns here as well.
+
+## Non-Requirements
+
+> Call out things you don't want to discuss in detail during this review here, to help focus the conversation. This can
+> include things you may build in the future based off this design, but don't wish to discuss in detail, in which case
+> it may also be wise to explicitly list that extensibility in your design is a requirement.
+>
+> This list can be high level and not detailed. It is to help focus the conversation on what you want to focus on.
+
+## Detailed design
+
+We want to introduce some new under `datahub/metadata-models/src/main/pegasus/com/linkedin/common/`.
+
+### `Tag` entity
+
+First we create a `TagMetadata` entity, which defines the actual tag-object.
+
+The URN of the tag consists for two parts: a category and a name. This gives us more flexibility as to how tags are use
+and reduces the risk of name collisions. An example of an `category` could be `sensitivity` and the name could be `pii`
+or `public`.
```

Comment:
I was also thinking about adding a tag `category` field. Right now I'm favoring allowing tags on tags, and using the tags applied to a given tag as its "category". This way we won't have to create two different grouping systems.

Reply:
Fixed as proposed.